src/components/Terms/Terms.react.js

```diff
@@ -22,6 +22,7 @@ class Terms extends Component {
 
   componentDidMount() {
      document.title = 'Terms and Conditions - SUSI.AI, Open Source Artificial Intelligence for Personal Assistants, Robots, Help Desks and Chatbots';
+     $('html, body').animate({ scrollTop: 0 }, 'fast');
```

Comment:
Refer https://github.com/fossasia/chat.susi.ai/pull/694 . It was to scroll to top in static pages. Madhavs PR also had a componentDIdMount - https://github.com/fossasia/chat.susi.ai/pull/692 and there were duplicate declarations of componentDidMount so refactored a bit